webapp/src/pages/Map.tsx

```diff
@@ -95,6 +95,7 @@ import {
   selectSelectedIntersection,
   setSelectedIntersectionId,
 } from '../generalSlices/intersectionSlice'
+import { evaluateFeatureFlags } from '../feature-flags'
 import { headerTabHeight } from '../styles/index'
```

Comment:
Theres a few unused imports and variables in this file we can clean up while we're in here